Documentation/Documentation/HealthDataStore.swift

```diff
@@ -24,7 +25,23 @@ public class HealthDataStore{
     }
     ````
     */
-    public func publicFunction(){
+    
+    func showAlert(_ VC : UIViewController, andMessage message: String , okClickHandler :@escaping(UIAlertAction) -> Void){
+        DispatchQueue.main.async {
+            let alert = UIAlertController(title: "APPLICATION_NAME", message: message , preferredStyle: UIAlertController.Style.alert)
+            alert.addAction(UIAlertAction(title: "Ok", style: UIAlertAction.Style.default, handler: okClickHandler))
+            alert.addAction(UIAlertAction(title: "Cancel", style: UIAlertAction.Style.cancel, handler: nil))
+            VC.present(alert, animated: true, completion: nil)
+        }
+    }
+    
+    /*
+     Call this function to add two numbers
+     */
```

Comment:
write the comment accurately.

Reply:
In next cl I will add the accurate comment.